.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**Description**
+Include a summary of the change, and link the associated issue (if applicable)
```

Comment:
Not sure how this is going to be different from the messages POD developers entered every time when committed changes. Maybe providing separate examples for newly submitted PODs and subsequent bug-fixing cases?

Reply:
The summary is ideally shorter than the entire list of commits. It could simply contain the scripts, files, and directories that were added/updated.

Reply:
@jkrasting and @wrongkindofdoctor, May I suggest that testing this template with a latest PR? Then we can point the later developers to a template with specific examples.